src/progress.c

```diff
@@ -213,9 +218,12 @@ void progress_next_step(const char *step_title, enum progress_type type)
 		debug("Please initialize the steps\n");
 		return;
 	}
+	debug("Current step name: %s\n", step_title);
+	debug("Current step number: %d (out of %d)\n", current_step + 1, total_steps);
 	if (current_step >= total_steps) {
-		debug("The current step (%d) is >= total steps (%d)\n", current_step, total_steps);
-		return;
+		debug("Warning: The current steps exceed the number of steps defined for the current operation\n");
```

Comment:
the current step exceeds the number of steps defined for the current operation

Reply:
I actually meant the "current steps", because what we define for each operation is the Total number of steps, so here it is saying that the current number of steps that have run exceed the amount defined